diff -ru openmpi-1.6.4.orig/ompi/contrib/vt/vt/extlib/otf/tools/otfshrink/otfshrink.cpp openmpi-1.6.4/ompi/contrib/vt/vt/extlib/otf/tools/otfshrink/otfshrink.cpp
--- openmpi-1.6.4.orig/ompi/contrib/vt/vt/extlib/otf/tools/otfshrink/otfshrink.cpp	2012-07-27 07:41:17.000000000 -0400
+++ openmpi-1.6.4/ompi/contrib/vt/vt/extlib/otf/tools/otfshrink/otfshrink.cpp	2013-02-25 12:59:21.000000000 -0500
@@ -54,7 +54,7 @@
 "      -V            show OTF version                                       \n",
 "      -i <file>     input file name                                        \n",
 "      -o <name>     namestub of the output file                            \n",
-"                    (default: "DEFAULT_OUTFILE")                           \n",
+"                    (default: " DEFAULT_OUTFILE ")                         \n",
 "      -l \"<list>\"   a list of processes in quotes                        \n",
 "                    to enable, i.e. keep in the copy,                      \n",
 "                    e.g. '-l \"1,2 4-8 3\",10 12-20'                       \n",
diff -ru openmpi-1.6.4.orig/ompi/contrib/vt/vt/extlib/otf/tools/vtf2otf/vtf2otf.c openmpi-1.6.4/ompi/contrib/vt/vt/extlib/otf/tools/vtf2otf/vtf2otf.c
--- openmpi-1.6.4.orig/ompi/contrib/vt/vt/extlib/otf/tools/vtf2otf/vtf2otf.c	2012-07-27 07:41:20.000000000 -0400
+++ openmpi-1.6.4/ompi/contrib/vt/vt/extlib/otf/tools/vtf2otf/vtf2otf.c	2013-02-25 13:00:03.000000000 -0500
@@ -41,7 +41,7 @@
 "      -h, --help    show this help message                                 \n",
 "      -V            show OTF version                                       \n",
 "      -o <file>     namestub of the output file                            \n",
-"                    (default: "DEFAULT_OUTFILE")                           \n",
+"                    (default: " DEFAULT_OUTFILE ")                         \n",
 "      -f <n>        max count of filehandles                               \n",
 "                    (default: 100)                                         \n",
 "      -n <n>        output stream count                                    \n",
@@ -167,7 +167,7 @@
 		printf( " no output file specified\n" );
 		exit(1);
 		*/
-		outputFile= strdup( DEFAULT_OUTFILE".otf" );
+		outputFile= strdup( DEFAULT_OUTFILE ".otf" );
 	}
 
 	if ( maxfilehandles < 1 ) {
